packages/vm/src/bin/module_size.rs

```diff
@@ -0,0 +1,46 @@
+use std::fs::File;
```

Comment:
I guess this file was supposed to be `bin/module_size.rs` not `src/bin/module_size.rs` right? The binaries (like benchs, examples, tests) are not in `src`. Anyways, this will change when turned into an example.

Reply:
No, this is a just way to compile an `bin` instead of a `lib`. Not sure what happens if I put it in `./bin` directly... wil try.

There are other ways of having mixed or multiple build artifacts with a single `Cargo.toml`, (i. e. having both `[lib]` and `[[bin]]` sections.
I like this one because it doesn't require changes to the `Cargo.toml`.

Reply:
> There are other ways of having mixed or multiple build artifacts with a single `Cargo.toml`, (i. e. having both `[lib]` and `[[bin]]` sections.

Yeah, this is the way I was referring to. It clearly separates the library from binaries build around it.